modules/service-fips.sh

```diff
@@ -50,6 +54,62 @@ fips_disable() {
     not_supported 'Disabling FIPS'
 }
 
+fips_update() {
+    local token="$1"
+    local bypass_prompt="$2"
+    local fips_configured=0
+
+    check_token "$FIPS_UPDATES_REPO_URL" "$token"
+
+    echo -n "Updating FIPS packages will take the system out of FIPS compliance."
```

Comment:
Please drop the `-n` from this echo, or else the next statement will be added right next to it:
```
Checking token... OK
Updating FIPS packages will take the system out of FIPS compliance.Do you want to proceed? [N/y] 
```

Reply:
ok. There will be an extra blank line when I remove -n. Is that OK?
Checking token... OK
Updating FIPS packages will take the system out of FIPS compliance.

Do you want to proceed? [N/y]

Reply:
Do you know what adds that extra line? `prompt_user()`? In any case, go ahead and remove `-n` please, it's used for when we want another subsequent `echo` to finish up a sentence, like in the case of reporting `OK` or `ERROR`.

Reply:
Removed -n. Fixed.